src/deepsparse/transformers/server/serve.py

```diff
@@ -0,0 +1,116 @@
+"""
+DeepSparse Inference Server for tasks supported by deepsparse.transformers'
+Pipeline module
+
+##########
+usage: Run DeepSparse Inference Server [-h] [--host HOST] [-p PORT]
+                                       [-w WORKERS]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --host HOST, -H HOST  The IP address of the hosted model
+  -p PORT, --port PORT  The port that the model is hosted on
+  -w WORKERS, --workers WORKERS
+                        The number of workers to use for uvicorn
+
+##########
+Example:
+1) From project root
+$deepsparse.transformers.serve --model-path <MODEL-PATH>
+"""
+import concurrent.futures
```

Comment:
let's refactor to `from concurrent.futures import Future` since we're only using this as a return type and nothing else here